docs/src/main/sphinx/develop/client-protocol.md

```diff
@@ -9,50 +9,47 @@ It can also be used to implement your own client, if necessary.
 ## HTTP methods
 
 - A `POST` to `/v1/statement` runs the query string in the `POST` body,
-  and returns a JSON document containing the query results.  If there are more
+  and returns a JSON document containing the query results. If there are more
   results, the JSON document contains a `nextUri` URL attribute.
 - A `GET` to the `nextUri` attribute returns the next batch of query results.
 - A `DELETE` to `nextUri` terminates a running query.
 
 ## Overview of query processing
 
 A Trino client request is initiated by an HTTP `POST` to the endpoint
-`/v1/statement`, with a `POST` body consisting of the SQL query string.
-The caller may set various {ref}`client-request-headers`. The headers are
-only required on the initial `POST` request, and not when following the
-`nextUri` links.
+`/v1/statement`, with a `POST` body consisting of the SQL query string. The
+caller may set various [](client-request-headers). The headers are only
+required on the initial `POST` request, and not when following the `nextUri`
+links.
 
-If the client request returns an HTTP 502, 503 or 504, that means there was
-intermittent problem processing request and the client should try again
-in 50-100 milliseconds. Trino does not generate those codes by itself
-but those can be generated by gateways/load balancers in front of Trino.
-Any HTTP status other than 502, 503, 504 or 200 means that query processing
-has failed.
+If the client request returns an HTTP 502, 503, or 504, that means there was
+intermittent problem processing request and the client should try again in
+50-100 milliseconds. Trino does not generate those codes by itself but those can
```

Comment:
1/ Do we actually write milliseconds out? We don't write seconds out I think?

2/ itself, but

Reply:
Updated